_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Pycon Pune 2017: A wonderful Python conference"
+date: "2017-03-14 18:18:02 +0530"
+tag: python conference
+---
+
+![pycon_pune_group_photo]({{
+site.url}}/assets/images/pycon_pune_group_photo.jpg)
+
+## tl;dr
+
+The conference is worth to attend if you are a student, programmer or a
+hobbyist.  You are swag-hungry then don't expect much as a swag from this
+conference. You are a Devsprint lover, then this conference has the coolest
+Devsprint. A great number of keynotes speackers are invited for this conference.
+
+
+## Detailed Experience
+
+Because I was volunteering for this conference I reached Pune one day earlier
+from the conference days. The volunteer meeting was happening at Reserved-bit.
+
+### Volunteer
+
+#### Reserved-bit
+
+[Reserved-bit][reservedbit] is the best hackerspace I have ever come across. It
+has a large collection of programmable boards. You will find boards like
+[Raspberry Pi](https://en.wikipedia.org/wiki/Raspberry_Pi), [Bana
+Pi](https://en.wikipedia.org/wiki/Banana_Pi),
+[Dragonboard](https://developer.qualcomm.com/hardware/dragonboard-410c),
+[Bigalbon](https://www.bigalsmods.com/product-p/1001.htm),
+[BBC-microbit](https://en.wikipedia.org/wiki/Micro_Bit) and the 3D printer. More
+on, this space has a great collection of books on Compilers and Embedded
+programming. I managed to found few on Open source too. The owners are great
+hackers. You will love to interact with hacker [Siddhesh
+Poyarekar](https://siddhesh.in/). Hacker [Nisha
+Poyarekar](https://twitter.com/nisha_poyarekar) is volunteering the [PyLadies
+community](https://www.meetup.com/PyLadies-Pune/) at Pune.
+
+
+#### Pune to Mumbai
+
+I spent my half day in this space. I got the responsibility of picking one of
+the keynote speakers landing at Mumbai airport midnight. To be frank, estimation
+of Google map between Pune to Mumbai is wrong. It showed nearly 2 hours but it
+took almost 4.5 hours to reach Mumbai. Few more minutes to reach the airport.
+The road is impressively smooth. You will encounter beautiful mountains of
+[Lonavla](https://en.wikipedia.org/wiki/Lonavla). The task of moving from Pune
+to Mumbai airport, receive Katie and come back to Pune completed in almost 13
+hours. I left from Pune around 4.30 PM and came back nearby 5 AM next day early
+morning.
+
+#### Illness during conference
+
+Because I did the huge amount of traveling at that night, I was unable to get
+enough sleep. Such tiredness resulted as an eye infection. I managed to attend
+the first day of the conference, but I was not in a condition to attend the
+second day. Treatment of local doctor healed me 2 days and then I was able to
+take part into Dev sprints.
```

Comment:
"into" should just be "in" here, and "Dev" is oddly capitalized - "Devsprint" scans well, since it's an unusual word that could easily be a proper noun, but "Dev sprint" is odd-looking in the middle of a sentence, and inconsistent with the single-word version you used earlier.